src/backend/joanie/tests/test_models_product.py

```diff
@@ -62,3 +62,20 @@ def test_generate_invoice(self):
         order.refresh_from_db()
         now = timezone.localtime(timezone.now())
         self.assertTrue(order.invoice_ref.startswith(now.strftime("%Y")))
+
+    def test_deliver_certificate(self):
```

Comment:
```suggestion
    def test_model_order_generate_certificate(self):
```

We have the habit to add context to the test name. In this way, it's easier to run only targeted test.
e.g:
```
pytest -k test_model_order
```